app/routes/notification.js

```diff
@@ -0,0 +1,23 @@
+import Route from '@ember/routing/route';
+
+export default class NotificationRoute extends Route {
+    model() {
+        return [{
+            title: "Hello User",
+            description: "Welcome to RDS"
+        },
+        {
+            title: "Your PR is pending",
+            description: "Your PR for my website is pending. Kindly check"
+        },
+        {
+            title: "Workshop on Dynamic Programming",
+            description: "Hey user we, are organizing a workshop on dynamic programming from next week. You will find the link soon. Kindly join."
+        },
+        {
+          title: "Merge Successful",
+          description: "Your PR is Successfully Merged. Thank you for your contribution"
+        }
+        ];
+      }
```

Comment:
Is there any other way to refactor this hard coded data by  moving hard coded data to another place and fetch from there.